app/electron/plugin-management.ts

```diff
@@ -378,6 +384,8 @@ function validateArchiveURL(archiveURL) {
   const githubRegex = /^https:\/\/github\.com\/[^/]+\/[^/]+\/(releases|archive)\/.*$/;
   const bitbucketRegex = /^https:\/\/bitbucket\.org\/[^/]+\/[^/]+\/(downloads|get)\/.*$/;
   const gitlabRegex = /^https:\/\/gitlab\.com\/[^/]+\/[^/]+\/(-\/archive|releases)\/.*$/;
+  // For testing purposes, we allow localhost URLs.
```

Comment:
Could this be enabled under test only?

`if (process.env.NODE_ENV === 'test') { `